bin/oneoff/populate_pilots

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+
+# Simple script for importing pilots that were hard-coded in
+# /models/experiments/experiment.rb into the newly created pilots table.
+
+require_relative '../../dashboard/config/environment'
+
+rows_added = 0
+Experiment::PILOT_EXPERIMENTS.each do |pilot|
```

Comment:
Is this list going to stay in the `Experiment` model when you're done here? Might be nice to have an explicit record of what was imported if it might disappear/change. FWIW, we're working on like identical things right now, and I did this via migration. Not sure if we have a preference in how this type of thing is done generally:

https://github.com/code-dot-org/code-dot-org/pull/40343/files

Reply:
I originally did it via migration, but that means that every time someone creates a new database from scratch, they will get these values.  For the pilots list, it seemed more that I wanted to import the list at this moment in time (and am ok with a new database having no values in the pilots table).

I considered copying the table for the reason you described but kind of liked that it directly referenced the existing table to drive home that this script is about importing the hard-coded list as it existed at the time it was run.  I guess copying the list would have the benefit that someone could later run this script if they just wanted to create a few pilots.  Hmmm.

Reply:
After I add the UI, you'll be able to go to the admin page to add a pilot.  (In general, a db from scratch should contain nothing, right?  We seed courses and scripts through a seeding process, but pilots didn't seem to be a core enough scenario to include in the seeding process?)

Reply:
i'm not sure i agree. i'd argue that since we sometimes have code paths tied to specific pilots (e.g. the time spent pilot), it should be as easy as possible for any engineer to demo and test that code. the number of code paths we already have that require extra setup to test (e.g. standards reporting, LevelBuilder functionality, deep learning, etc.) is bad enough, so i'd rather not see us add another. especially one with no easy "seeding" process ("you want to test time spent? let me look in the code for the name of the pilot you need to create").

maybe that's just the nature of the beast though. either way, i'm fine with leaving this as-is and we can address if it becomes a pain point.

Reply:
another option we could consider as a followup would be to add a flag to `locals.yml` that would make the check for whether an experiment is enabled always return `true`...

Reply:
You raise interesting points, let's discuss at our next stand-up.